docs/README.md

```diff
@@ -0,0 +1,23 @@
+Release Notes 
```

Comment:
we should add formating, headers, etc
similar to what we have in mockintosh 
https://github.com/up9inc/mockintosh/blob/main/docs/Changelog.md